build/master/html/paris.html

```diff
@@ -0,0 +1,1348 @@
+<li class="toctree-l5"><a class="reference internal" href="mongodb/actions/collection.find/#working-with-bson">Working with BSON</a></li>
+</ul>
+</li>
+<li class="toctree-l4"><a class="reference internal" href="mongodb/actions/collection.find/#examples">Examples</a><ul>
+<li class="toctree-l5"><a class="reference internal" href="mongodb/actions/collection.find/#finding-all-documents-in-a-collection">Finding All Documents in a Collection</a></li>
+<li class="toctree-l5"><a class="reference internal" href="mongodb/actions/collection.find/#finding-documents-matching-user-input">Finding Documents Matching User Input</a></li>
+<li class="toctree-l5"><a class="reference internal" href="mongodb/actions/collection.find/#finding-documents-matching-a-regular-expression">Finding Documents Matching a Regular Expression</a></li>
+<li class="toctree-l5"><a class="reference internal" href="mongodb/actions/collection.find/#sort-and-limit-results">Sort and Limit Results</a></li>
+</ul>
+</li>
+</ul>
+</li>
+<li class="toctree-l3"><a class="reference internal" href="mongodb/actions/collection.findOne/">collection.findOne()</a><ul>
+<li class="toctree-l4"><a class="reference internal" href="mongodb/actions/collection.findOne/#definition">Definition</a></li>
+<li class="toctree-l4"><a class="reference internal" href="mongodb/actions/collection.findOne/#behavior">Behavior</a><ul>
+<li class="toctree-l5"><a class="reference internal" href="mongodb/actions/collection.findOne/#working-with-bson">Working with BSON</a></li>
+</ul>
+</li>
+<li class="toctree-l4"><a class="reference internal" href="mongodb/actions/collection.findOne/#examples">Examples</a><ul>
+<li class="toctree-l5"><a class="reference internal" href="mongodb/actions/collection.findOne/#finding-the-first-document-in-a-collection">Finding the First Document in a Collection</a></li>
+<li class="toctree-l5"><a class="reference internal" href="mongodb/actions/collection.findOne/#finding-a-document-matching-user-input">Finding a Document Matching User Input</a></li>
+<li class="toctree-l5"><a class="reference internal" href="mongodb/actions/collection.findOne/#finding-a-document-matching-a-regular-expression">Finding a Document Matching a Regular Expression</a></li>
+</ul>
+</li>
+</ul>
+</li>
+<li class="toctree-l3"><a class="reference internal" href="mongodb/actions/collection.count/">collection.count()</a><ul>
+<li class="toctree-l4"><a class="reference internal" href="mongodb/actions/collection.count/#definition">Definition</a></li>
+<li class="toctree-l4"><a class="reference internal" href="mongodb/actions/collection.count/#examples">Examples</a><ul>
+<li class="toctree-l5"><a class="reference internal" href="mongodb/actions/collection.count/#counting-all-documents-in-a-collection">Counting All Documents in a Collection</a></li>
+<li class="toctree-l5"><a class="reference internal" href="mongodb/actions/collection.count/#co
```

Comment:
may be:   monitoring => surveillance
but the Frenchs have a lot of those english words.
Interesting enough, Québécois do too, however often different words.

Reply:
So convenient. I've changed to surveillance, though 'monitoring' seems relatively popular on french informatique websites as well. 🤷‍♀️